### PR TITLE
ApiDoc Group

### PR DESCRIPTION
**Pull request by librarianmage**
[//]: # (Note: See http://habitica.wikia.com/wiki/Using_Habitica_Git#Pull_Request for more info)

[//]: # (Put Issue # or URL here, if applicable. This will automatically close the issue if your PR is merged in)
Part of #8087

### Changes
[//]: # (Describe the changes that were made in detail here. Include pictures if necessary)

[//]: # (Put User ID in here - found in Settings -> API)

----
UUID:

### CODE REVIEW

**Comment by TheHollidayInn** [comment]
I think this just need success and param examples on each route and then it is good.

File: website/common/locales/en/groups.json [hunk 1]
```diff
@@ -102,6 +102,7 @@
   "messageSentAlert": "Message sent",
   "pmHeading": "Private message to <%= name %>",
   "pmsMarkedRead": "Your private messages have been marked as read",
+  "possessiveParty": "<%= name %>'s Party",
```
Comment (paglias) [inline website/common/locales/en/groups.json]: are these new strings expected?

**Review by paglias — CHANGES_REQUESTED** [review]

**Comment by librarianmage** [comment]
They were there when I merged develop in, so I have no idea

**Comment by paglias** [comment]
yeah, it's strange, can you remove them?

**Comment by TheHollidayInn** [comment]
I added those in when merging as well. I thought they were from some new PR.

**Comment by paglias** [comment]
now it's even stranger, with a full file differing. Can you close this pr and open a new one from a clean develop branch?

**Comment by TheHollidayInn** [comment]
I see this issue often with merging locals.

**Comment by librarianmage** [comment]
Could there be something wonky with the line endings in the locales?

**Comment by TheHollidayInn** [comment]
Yea I definitely think it is with white characters. Probably my fault, tbh

**Comment by librarianmage** [comment]
New PR located at #8522

**Pull request closed without merging**